### ISSUE TITLE
Uri.queryParameters return map, where value include "=" char

### ISSUE THREAD

**Issue by kaserdan** [chRyNaN/uri#1]
Uri.queryParameters returns key-value map, but value includes also the "=" char.

```
 val queryParamValue = Uri.fromString(uriString = "www.example.com?key=value").queryParameters()["key"]
 // queryParamValue is: "=value"
 // expected: "value"
```

**Comment by chRyNaN** [comment]
Good catch! I'll take a look at this shortly and see if I can get a fix out today.

**Comment by chRyNaN** [comment]
This should be fixed in [Release 0.3.2 (5)](https://github.com/chRyNaN/uri/releases/tag/0.3.2)
